examples/Training/java/src/training/HowToUseTables.java

```diff
@@ -114,6 +132,23 @@ private void createTable()
             table.addData(newRow);
             OriginalFile file = table.getOriginalFile(); // if you need to interact with the table
             file = new OriginalFileI(file.getId(), false);
+
+            DataManagerFacility dm = gateway.getFacility(DataManagerFacility.class);
+
+            FileAnnotation annotation = new FileAnnotationI();
+            annotation.setFile(file);
+            annotation.setNs(omero.rtypes
+                    .rstring(omero.constants.namespaces.NSBULKANNOTATIONS.value));
+            annotation = (FileAnnotation) dm.saveAndReturnObject(ctx, annotation);
```

Comment:
Tested it locally, looks like works even without this line! And it makes sense in a bulk workflow. What do you suggest @jburel, @mtbc's comment on bulk workflows definitely makes sense.! If we are doing it this way should we update our docs page in a similar way as well?